.speakeasy/workflow.yaml

```diff
@@ -8,3 +7,6 @@ targets:
     first-target:
         target: csharp
         source: openapi
+        publish:
+            nuget:
+                apiKey: $nuget_api_key
```

Comment:
This looks promising -- it's the bit I had originally forgotten when I hadn't used the CLI for the setup. 🤞